_docs/getting-started/windows.md

```diff
@@ -91,10 +94,11 @@ For VS15 and VS17 users with GitHub Desktop:
 
 ### Python Configuration
 
-You will find the Caffe2 binary in `$USER\Source\Repos` (if that's where you put the caffe2 source) `\caffe2\build\caffe2\binaries\Release`
+You will find the Caffe2 binary in `$USER\Source\Repos` (if that's where you put the caffe2 source) `\pytorch\build\caffe2\python`
 
-To get python to recognize the DLL, rename `caffe2_pybind11_state.dll` from `.dll` to `.pyb` and copy it to Python's DLL folder `$USER\AppData\Local\Continuum\Anaconda2\DLLs`. If you're not using Anaconda, then look for it in your Python27 or python-2713 folder.
+Copy `caffe2_pybind11_state.pyd` to Python's DLL folder `$USER\AppData\Local\Continuum\Anaconda2\DLLs`. If you're not using Anaconda, then put it in your Python27 or python-2713 folder.
```

Comment:
Also, @pjh5, any case where `caffe2_pybind11_state.dll` (instead of `.pyd`) would be generated now?

Reply:
I'm not sure about this